src/cljs/zetawar/views.cljs

```diff
@@ -74,9 +81,20 @@
                    (not @(subs/unit-can-act? conn q r)))
 
               ;; Unit selected and tile is a valid attack or move target
+              ;; However, selected cannot repair others
+              (and @(subs/unit-selected? conn)
```

Comment:
I think you can combine the 'and's on lines 85 and 93 by adding '@(subs/selected-can-repair-other? conn)' into the 'not' + 'and' currently on line 96 and moving into the higher level and starting on line 85 (hope that makes sense + is correct :smile:).